Backend/WebApi/Program.cs

```diff
@@ -88,14 +108,18 @@
 
     // Add web UIs to interact with the document
     // Available at: http://localhost:<port>/swagger
-    app.UseSwaggerUi(c => c.CustomInlineStyles = darkStyles);
+    app.UseSwaggerUi(c =>
+    {
+        c.CustomInlineStyles = darkStyles;
+    }
+    );
 
     // Add ReDoc UI to interact with the document
     // Available at: http://localhost:<port>/redoc
-    app.UseReDoc(options =>
+    /*app.UseReDoc(options =>
```

Comment:
Czemu wywalasz redoca?

Reply:
Unhandled exception. System.NotSupportedException: This UI does not support multiple documents per UI: Do not use '{documentName}' placeholder in DocumentPath or Path.
   at Microsoft.AspNetCore.Builder.NSwagApplicationBuilderExtensions.UseSwaggerUiWithDocumentNamePlaceholderExpanding(IApplicationBuilder app, SwaggerUiSettingsBase settings, Action`2 register, Func`2 registerMultiple)
   at Microsoft.AspNetCore.Builder.NSwagApplicationBuilderExtensions.UseReDoc(IApplicationBuilder app, Action`1 configure)

Wyrzuca mi taki błąd